omnibus/config/software/datadog-agent.rb

```diff
@@ -154,7 +156,8 @@
 
   # Security agent
   unless windows?
-    command "invoke -e security-agent.build --major-version #{major_version_arg}", :env => env
+    build_tags = linux? ? "" : ""
```

Comment:
Is this an error? If I'm looking right, we always use `""` as `build_tags`?

Reply:
This will go away, since we won't link with the OpenSCAP library by default.